en-US/foundation/index.lang

```diff
@@ -0,0 +1,142 @@
+## NOTE: demo page at https://www-dev.allizom.org/foundation
+## URL: https://www-dev.allizom.org/%LOCALE%/foundation
+
+
+;The Mozilla Foundation
+The Mozilla Foundation
+
+
+;Cybersecurity
+Cybersecurity
```

Comment:
I see this in bedrock/foundation/templates/foundation/index.html but it doesn't seem used anywhere?